.vale.ini

```diff
@@ -14,6 +14,12 @@ Vocab = konflux
 
 BasedOnStyles = RedHat, Konflux
 
+# Voice consistency rules with recommended levels
```

Comment:
Those level are configured in the rules themselves, why do we override configuration here, instead of changin level in rules?